scripts/publish-release

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+
+. "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/scriptLib"
+
+printMessage "PUBLISH RELEASE ARTIFACTS"
+sbt +publishSigned
```

Comment:
If we will signing artifacts in Bintray, need to change
```suggestion
sbt +publish
```

Reply:
Correct. We don't sign on the machine publishing the artifacts.